lib/api/createJob.js

```diff
@@ -0,0 +1,31 @@
+var fs = require("fs");
+var path = require('path');
+var _ = require('underscore');
+var authHelper = require("../util/auth")
+
+/**
+ *  Create jenkins job
+ */
+module.exports = (auth, jobArgs, callback) => {
+    var options = authHelper.createJenkinsOptions(auth.url, auth.user, auth.password);
+    var jenkins = require('jenkins')(options);
+    var fileLocation = path.join(__dirname, '..', '..', 'templates', 'standard.xml');
```

Comment:
That's just plain template to create job. We using jenkinsfiles but we would need some simple xml file to create job that would have only giturl and branch configured. Rest would come from Jenkinsfile itself. 

I investigated Jenkinfile's automatic job creation and it's really powerful, but for our internal needs we would need to create job by calling this.